packages/flutterfire_ui/lib/src/i10n/lang/hu.dart

```diff
@@ -0,0 +1,263 @@
+    this.signInWithPhoneButtonText = 'Bejelentkezés telefonszámmal',
+    this.signInWithGoogleButtonText = 'Google bejelentkezés',
+    this.signInWithAppleButtonText = 'Apple bejelentkezés',
+    this.signInWithTwitterButtonText = 'Twitter bejelentkezés',
+    this.signInWithFacebookButtonText = 'Facebook bejelentkezés',
+    this.phoneVerificationViewTitleText = 'Add meg a telefonszámod',
+    this.verifyPhoneNumberButtonText = 'Következő',
+    this.verifyCodeButtonText = 'Ellenőrzés',
+    this.verifyingPhoneNumberViewTitle = 'Add meg az SMS kódot',
+    this.unknownError = 'Ismeretlen hiba történt',
+    this.smsAutoresolutionFailedError =
+        'Nem sikerült az SMS kódot kinyerni. Kérlek add meg a kódot manuálisan',
+    this.smsCodeSentText = 'SMS kód elküldve',
+    this.sendingSMSCodeText = 'SMS kód küldése...',
+    this.verifyingSMSCodeText = 'SMS kód ellenőrzése...',
+    this.enterSMSCodeText = 'Add meg az SMS kódot',
+    this.emailIsRequiredErrorText = 'Email cím megadása kötelező',
+    this.isNotAValidEmailErrorText = 'Helyes email címet adj meg',
+    this.userNotFoundErrorText = 'A fiók nem létezik',
+    this.emailTakenErrorText = 'Ezzel az email címmel már regisztráltak',
+    this.accessDisabledErrorText =
+        'Ez a fiók átmenetileg le van tiltva',
+    this.wrongOrNoPasswordErrorText =
+        'Érvénytelen jelszó vagy a felhasználónak nincs jelszava',
+    this.signInText = 'Bejelentkezés',
+    this.registerText = 'Regisztráció',
+    this.registerHintText = 'Nincs fiókod?',
+    this.signInHintText = 'Már van fiókod?',
+    this.signOutButtonText = 'Kijelentkezés',
+    this.phoneInputLabel = 'Telefonszám',
+    this.phoneNumberInvalidErrorText = 'Érvénytelen telefonszám',
+    this.phoneNumberIsRequiredErrorText = 'Telefonszám megadása kötelező',
+    this.profile = 'Profil',
+    this.name = 'Név',
+    this.deleteAccount = 'Fiók törlése',
+    this.passwordIsRequiredErrorText = 'Jelszó megadása kötelező',
+    this.confirmPasswordIsRequiredErrorText = 'Jelszó megerősítése',
+    this.confirmPasswordDoesNotMatchErrorText = 'Jelszavak nem egyeznek',
+    this.confirmPasswordInputLabel = 'Jelszó ismét',
+    this.forgotPasswordButtonLabel = 'Efelejtett jelszó?',
+    this.forgotPasswordViewTitle = 'Elfelejtett jelszó',
+    this.resetPasswordButtonLabel = 'Jelszó visszaállítása',
+    this.verifyItsYouText = 'Igazold magad',
+    this.differentMethodsSignInTitleText =
+        'Válassz az alábbi bejelentkezés lehetőségek közül',
+    this.findProviderForEmailTitleText = 'Add meg az email címed a folytatáshoz',
+    this.continueText = 'Folytatás',
+    this.countryCode = 'Országkód',
+    this.codeRequiredErrorText = 'Országkód megadása kötelező',
+    this.invalidCountryCode = 'Érvénytelen országkód',
+    this.chooseACountry = 'Válassz országot',
+    this.enableMoreSignInMethods = 'Több bejelentkezési módszer engedélyezése',
+    this.signInMethods = 'Bejelentkezési módszerek',
```

Comment:
i'd suggest using 'Bejelentkezési lehetőségek' to stay consistent with the above